lib/remote-model.js

```diff
@@ -150,11 +150,11 @@ class RemoteModel extends LocalModel {
         const isHttps = (new URL(url).protocol === 'https:');
 
         if (isHttps) {
-            const HttpsProxyAgent = require('https-proxy-agent');
+            const { HttpsProxyAgent } = require('https-proxy-agent');
             const agent = new HttpsProxyAgent(options);
             return { https: agent };
         } else {
-            const HttpProxyAgent = require('http-proxy-agent');
+            const { HttpProxyAgent } = require('http-proxy-agent');
```

Comment:
The proxy agent constructors here don't seem to match the changed API introduced in 6.0.0 so calls through a proxy are most likely broken. Tests might not be working properly if they're passing on this.

https://github.com/TooTallNate/proxy-agents/blob/main/packages/https-proxy-agent/CHANGELOG.md#600
https://github.com/TooTallNate/proxy-agents/blob/main/packages/http-proxy-agent/CHANGELOG.md#600